client/client.go

```diff
@@ -414,6 +414,24 @@ func (c *Client) Info() (map[string]interface{}, error) {
 	return hash, nil
 }
 
+func (c *Client) Queues() (map[string]interface{}, error) {
+	hash, err := c.Info()
+	if err != nil {
+		return nil, err
+	}
+
+	queues, ok := hash["faktory"].(map[string]interface{})["queues"].(map[string]interface{})
```

Comment:
You need to verify these typecasts. Faktory should never panic or crash.

Reply:
You still have two typecasts together here with no error checking.

Reply:
I have split them up now.